src/ontology/mondo-edit.obo

```diff
@@ -547007,6 +547007,14 @@ property_value: IAO:0000233 https://github.com/monarch-initiative/mondo/issues/5
 property_value: IAO:0000233 https://github.com/obophenotype/upheno/issues/871
 is_symmetric: true
 
+[Typedef]
+id: curated_content_resource
+name: curated content resource
+def: "A curated content resource is a web-resource that contains curated content about the ontology term." []
+comment: This property is likely to change the ID. Please do not use in production systems.
```

Comment:
What does `This property is likely to change the ID` mean? Change which ID?

Reply:
id: curated_content_resource

is likely to be swapped to an official OBO id, like 

```
OMO:0002005
```

Its just to painful to add these relations to OMO right now, and we want to move on a bit!